pkg/jaeger/store/find_traces.go

```diff
@@ -36,13 +36,19 @@ func findTraces(ctx context.Context, builder *Builder, conn pgxconn.PgxConn, q *
 
 func scanTraces(rows pgxconn.PgxRows) ([]*model.Trace, error) {
 	traces := ptrace.NewTraces()
+	spansBinaryTags := map[int64]*spanBinaryTags{}
```

Comment:
Instead of tracking span with binary tag why can't we just iterate over final jaeger trace and decode if the string type is matching base64 prefix?

Reply:
We are already iterating them when we decode from the database. 

I didn't want to have to do it twice, per every span, span process and span logs.

WDYT? Am I being too paranoid